File: iterableapi/src/main/java/com/iterable/iterableapi/IterableDecryptionFailureHandler.java
package com.iterable.iterableapi;

/**
 * Interface for handling decryption failures
 */
public interface IterableDecryptionFailureHandler {
    /**
     * Called when a decryption failure occurs
     * @param exception The exception that caused the decryption failure
     */
    void onDecryptionFailed(Exception exception);
} 

File: iterableapi/src/main/java/com/iterable/iterableapi/IterableDecryptionFailureHandler.kt
package com.iterable.iterableapi

/**
 * Interface for handling decryption failures
 */
interface IterableDecryptionFailureHandler {
    /**
     * Called when a decryption failure occurs
     * @param exception The exception that caused the decryption failure
     */
    fun onDecryptionFailed(exception: Exception)
}